includes/board.h

```diff
@@ -30,4 +32,6 @@ class Board {
     void removeFilledRows(int rowStart); // remove filled rows
 
     void setPiece(std::vector<int> rows, std::vector<int> cols, int color);
+
+    void draw();
```

Comment:
assuming this purposely a stub for now. I suggest you add TODOs to make reviewing this easier for me.